packages/components/src/List/ListComposition/index.js

```diff
@@ -18,4 +19,7 @@ export default {
 	VList,
 };
 
-export { useCollectionSort };
+export const hooks = {
+	useCollectionSort,
+	useDisplayMode: useLocalStorage,
```

Comment:
What's the purpose to export it?

Reply:
To enforce the local storage behavior on display mode.
In project, we will use this hook if we want to control the display mode, adding automatically the local storage behavior